### ISSUE TITLE
Cross Validation question

### ISSUE THREAD

**Issue by jarlva** [EpistasisLab/tpot#799]
(This is a question, not issue). 
It's my understanding that tpot takes a CV argument when it does it's evaluation/optimization. if that's true, I'm not sure why I see in the examples train/test split/scoring. Wouldn't it be more accurate/efficient just to cross validate/score against the whole feature/labels data without having to split it to train/test?

**Comment by weixuanfu** [comment]
Though average CV scores of evaluated pipelines are used during optimization for prevent overfitting on training set, in our practices/examples, the test set was used for calculating holdout score to examine whether the overfitting happened.

**Comment by jarlva** [comment]
So, it's safe to allocate most of the dataset to train and a small portion (~10%) to the final prediction/final verification, right?

**Comment by weixuanfu** [comment]
It depends. If 10% of whole dataset is bigger enough for validation, it should be OK.
